pkg/config/setup/config.go

```diff
@@ -106,6 +106,15 @@ const (
 	// DefaultLogsSenderBackoffRecoveryInterval is the default logs sender backoff recovery interval
 	DefaultLogsSenderBackoffRecoveryInterval = 2
 
+	// DefaultLogsLegacyMatchThreshold is deprecated and used for legacy auto multiline detection.
+	DefaultLogsLegacyMatchThreshold = 0.48
+
+	// DefaultLogsLegacyMatchTimeout is deprecated and used for legacy auto multiline detection.
+	DefaultLogsLegacyMatchTimeout = 30
+
+	// DefaultLogsLegacySampleSize is deprecated and used for legacy auto multiline detection.
+	DefaultLogsLegacySampleSize = 500
+
```

Comment:
If we use `IsConfigured` in `comp/logs/agent/config/integration_config.go`, do we need those const ?